tutorials/overleaf/index.mdx

```diff
@@ -0,0 +1,203 @@
+    ```
+    ssh root@<YOUR-INSTANCE-IP>
+    ```
+2. Use `git` to downlaoad the Overleaf code repoisitory.
+    ```
+    git clone https://github.com/overleaf/toolkit.git /opt/overleaf-toolkit
+    ```
+3. Enter the Overleaf `overleaf-toolkit` directory.
+    ```
+    cd /opt/overleaf-toolkit
+    ```
+4. Run the following command to copy the configuration files
+    ```
+    ./bin/init
+    ```
+5. Download and start the Overleaf container and its requirements
+    ```
+    ./bin/up
+    ```
+    <Message type="tip">
+      * If a web server is already running on port 80, be sure to stop it before executing the above command.
+      * Dowloading and configuring the application for the first start may take some time.
+    </Message>
+6. Stop the application by pressing `CTRL+C`.
+7. Open the file `./config/overleaf.rc` in a text editor and edit the `NGINX_HTTP_PORT=80` line . Set the value to `NGINX_HTTP_PORT=8877`, then save the file and exit the editor.
+
+## Configuring a systemd script
+
+Once Overleaf runs correctly, you can create a systemd daemon service.
+
+1. Create and open the `/etc/systemd/system/overleaf.service` file in a text editor of your choice and paste the following content into it:
+    ```
+    [Unit]
+    Description=Overleaf Community Edition
+
+    [Service]
+    User=root
+    WorkingDirectory=/opt/overleaf-toolkit
+    ExecStart=/opt/overleaf-toolkit/bin/up
+    Restart=always
+    RestartSec=10
+
+    [Install]
+    WantedBy=multi-user.target
+
+    [Unit]
+    Description=Overleaf Community Edition
+    
+    [Service]
+    User=root
+    WorkingDirectory=/opt/overleaf-toolkit
+    ExecStart=/opt/overleaf-toolkit/bin/up
+    Restart=always
+    RestartSec=10
+    
+    [Install]
+    WantedBy=multi-user.target
+    ```
+2. Save the file and exit the text editor.
+3. Reload systemd by runnning the following command:
+    ```
+    systemctl daemon-reload
+    ```
+4. Enable the `overleaf.service` to be started on reboot:
+    ```
+    systemctl enable overleaf.service
+    ```
+5. Start the Overleaf service:
+    ```
+    systemctl start overleaf.service
+    ```
+6. Check the status of the Overleaf service:
+    ```
+    systemctl status overleaf.service
+    ```
+
+## Installing LaTeX packages
+
+The LaTeX packages can now be installed, using TeX Live.
+1. Run the following command to launch the Texlive manager update:
+    ```
+    docker exec Overleaf Community Edition tlmgr update --self
+    ```
+2. Run the following command to lauch the full `texlive` installation:
+    ```
+    docker exec Overleaf Community Edition tlmgr install scheme-full
+    ```
+    <Message type="note">
+      The installation of `texlive` may take a while.
+    </Message>
+
+## Configuring Nginx
+
+This tutorial uses [NGINX](https://nginx.org/en/) with [Let's Encrypt](https://letsencrypt.org/) and [CertBot](https://certbot.eff.org/) as an SSL enabled reverse proxy to secure the connection between the client and our server.
```

Comment:
Is the all-caps correct for Nginx? I don't know how it should be written.

Reply:
They usually write it NGINX but I can change it if it bothers you

Reply:
No it's fine